.gitignore

```diff
@@ -27,4 +27,4 @@ out
 *.sublime-project
 *.sublime-workspace
 build-local.properties
-
+.checkstyle
```

Comment:
This seems like it shouldn't be needed... I don't think it is in kaazing/gateway and they use a same checkstyle I think